addon/templates/components/inline-edit-field.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if isEditing}}
+  <form class="editable is-editing"
+    {{action "stopEditing" model on="submit"}}>
+
+    {{#if (hasBlock "inverse")}}
+      {{yield to="inverse"}}
+    {{else}}
+      {{input value=value}}
+    {{/if}}
+
+    <div class="save-and-cancel">
```

Comment:
I haven't had that be the case so far. Normally it's just the fact that you need a different type of input.

I'd like to explore solutions for customizing those buttons moving forward though.